components/Buttons/src/ColorThemer/MDCButtonColorThemer.h

```diff
@@ -61,4 +64,23 @@
 + (void)applyColorScheme:(nonnull id<MDCColorScheme>)colorScheme
                 toButton:(nonnull MDCButton *)button;
 
+/**
+ Applies a color scheme to theme to an MDCFlatButton.
+
+ @param colorScheme The color scheme to apply to @c flatButton.
```

Comment:
Include a link to the new, recommended API.

Reply:
Done.